### DISCUSSION TITLE
Adding parameters to winget command

### DISCUSSION THREAD

**Discussion by grayhunterstenn** [Romanitho/Winget-Install#84] [Q&A]
I would like to automate the installation of multiple versions of the Visual C++ Redistributable.  However, because of [an issue with winget](https://github.com/microsoft/winget-pkgs/issues/97053), I have to add the "--force" switch to the winget command.  How can I do that with Winget-Install?

**Comment by Romanitho** [comment]
https://github.com/Romanitho/Winget-Install#use-winget-native-parameters

so try with `--force` inside the appid argument
